### PR TITLE
First draft: Tell Grace's story about a journey to a new library.

### PR DESCRIPTION
**Pull request by Gottox**
Please note that I'm neither a technical writer nor am I a native speaker. This is just a quick summary of the experiences I had while doing some stuff with kube-rs. It would be awesome to get some help to get this into a state where it actually could help you pals to improve the experience when using async code.

Thanks for this project btw :)

### CODE REVIEW

File: src/vision/status_quo/grace_tries_new_libraries.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        match res {
+            Ok(o) => info!("reconciled {:?}", o),
+            Err(e) => warn!("reconcile failed: {}", Report::from(e)),
+        }
+    })
+    .await;
+```
+(Example code from taken from https://github.com/clux/kube-rs)
+
+So she takes the naiv approach to just convert that as follows:
```
Comment (nikomatsakis) [inline src/vision/status_quo/grace_tries_new_libraries.md:30]:
```suggestion
So she takes the naive approach to just convert that as follows:
```

**Review by nikomatsakis — COMMENTED** [review]
Thanks! Gave a quick read, found a typo, will read more later.

**Comment by Gottox** [comment]
Reading the article once more, I'm not sure if Grace is the right stereotype for this issue. I just see myself as a Grace. :thinking:

**Review by nikomatsakis — APPROVED** [review]
I left some FAQs but i'm inclined to merge this. Short and sweet.

**Review by nikomatsakis — APPROVED** [review]

**Comment by nikomatsakis** [comment]
@Gottox hope you don't mind -- I'm trying to move some PRs along, so i went ahead and added some FAQs. I'm going to merge but feel free to open a PR with edits if you have ideas for what to change or more to add!

**Comment by Gottox** [comment]
Feel free! I was rather unsure about the quality of my text. Hope I can help nontheless :+1:

**Pull request merged**